docs/diagrams/GraphSequenceDiagram.puml

```diff
@@ -0,0 +1,43 @@
+@startuml
+!include style.puml
+
+box "UI"
```

Comment:
"UI" and "Model" title are a bit hard to see. Maybe change the color of the boxes or the titles?